automatic/transifex-cli/Readme.md

```diff
@@ -0,0 +1,11 @@
+# [<img src="https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-packages@4c2d7c7377b05a3d0a2dd35cfc89ce0c77625051/icons/transifex-client.png" height="48" width="48" /> Transifex Client](https://chocolatey.org/packages/transifex-client)
+
+The Transifex CLI enables you to manage your translations within a project without the need of an elaborate UI system.
+
+You can use the command line tool to create new resources, map locale files to translations, and synchronize your Transifex project with your local repository.
+Translators and localization managers can use it to handle large volumes of translation files.
+The Transifex CLI can help to enable continuous integration workflows and can be run from CI servers like Jenkins and Bamboo.
+
+## Notes
+
+- This package uses the Go version of the Transifex CLI
```

Comment:
@pauby do you have an example of the notes you usually prefer to have added in this section to direct the user to where to open issues regarding bugs in the package?